### ISSUE TITLE
Bug report: google-cloud-sdk fish completion instructions are wrong

### ISSUE THREAD

**Issue by danielcompton** [caskroom/homebrew-cask#30847]
#### Description of issue

I installed the Google Cloud SDK cask and followed the install instructions for fish completions:

```
set fish_user_paths /usr/local/Caskroom/google-cloud-sdk/latest/google-cloud-sdk/bin
set -x MANPATH /usr/local/Caskroom/google-cloud-sdk/latest/google-cloud-sdk/help/man /usr/local/share/man /usr/share/man /opt/x11/share/man
```

then tried running `fish_update_completions`.

I got:

```
$ fish_update_completions
No paths specified
```

After removing those two lines from my fish config file, I was able to run `fish_update_completions` and got this correct looking result:

```
$ fish_update_completions
Parsing man pages and writing completions to /Users/myuser/.local/share/fish/generated_completions/
  3176 / 3176 : zic.8
```

It looks like the instructions in #9958 aren't valid anymore possibly?

I'm pretty sure that MANPATH should be set something like this, although it should also account for `MANPATH` not existing beforehand: https://github.com/fish-shell/fish-shell/issues/2090

```
set -x MANPATH /usr/local/Caskroom/google-cloud-sdk/latest/google-cloud-sdk/help/man $MANPATH
```

Currently there is no help directory or manpage subfolder at that location, so that won't work at the moment, and this causes the error `No paths specified`.

To summarise there are two problems here:

1. There is no manpage folder at the location suggested.
2. If there was, the MANPATH variable should be set in a way that respects the users existing MANPATH, and allows for MANPATH not previously existing.

**Comment by vitorgalvao** [comment]
Ping @daenney.

**Comment by daenney** [comment]
Hmm, that's annoying.

So looking at my own installation of Google Cloud SDK from Cask I can indeed see that the `help/man` no longer exists:

```
[I] 14:10:51 ~/h/C/g/l/google-cloud-sdk (stable) $ pwd
/Users/daenney/homebrew/Caskroom/google-cloud-sdk/latest/google-cloud-sdk

[I] 14:11:11 ~/h/C/g/l/google-cloud-sdk (stable) $ tree -d -L 2
.
├── bin
│   └── bootstrapping
├── lib
│   ├── googlecloudsdk
│   ├── surface
│   ├── third_party
│   └── tools
└── platform
    ├── bq
    ├── ext-runtime
    └── gsutil

11 directories
```

However, if I download one of the archives from https://cloud.google.com/sdk/downloads and extract it, they are there:

```
[I] 14:12:01 ~/D/google-cloud-sdk $ pwd
/Users/daenney/Downloads/google-cloud-sdk

[I] 14:12:02 ~/D/google-cloud-sdk $ tree -d -L 2
.
├── bin
│   └── bootstrapping
├── help
│   └── man
├── lib
│   ├── googlecloudsdk
│   ├── surface
│   ├── third_party
│   └── tools
└── platform
    ├── bq
    ├── ext-runtime
    └── gsutil

13 directories
```

So it seems that somehow these are getting lost when Cask is installing them, after that causing issues with the `MANPATH`. That said, Fish should ignore parts of the `MANPATH` that just don't exist so I'm confused as to why the `fish_update_completions` doesn't run at all. You're right though that the recommendations should state `set -x MANPATH .... $MANPATH`.

I'm not really a maintainer of this though, I just landed the patch last time to ensure Fish users could get correct completion since it wasn't working for me. I don't really know how to go about figuring out why the `help/` directory disappears.

**Comment by daenney** [comment]
Ah, the problem seem to stem from the following; the formula downloads the latest Cloud SDK from here: https://dl.google.com/dl/cloudsdk/release/google-cloud-sdk.tar.gz.

However, that bundle, contrary to the versioned bundles at https://cloud.google.com/sdk/downloads doesn't contain the `help/` directory.
